### ISSUE TITLE
library_if_available() is deprecated and removed in dev version of testthat

### ISSUE THREAD

**Issue by hadley** [gavinsimpson/analogue#11]
```
checking tests ... ERROR
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  Attaching package: 'analogue'
  
  The following object is masked from 'package:testthat':
  
      compare
  
  > 
  > ## Runs the tests in inst/tests
  > test_package("analogue")
  Error in eval(expr, envir, enclos) : 
    could not find function "library_if_available"
  Calls: test_package ... test_dir -> lapply -> FUN -> sys.source2 -> eval -> eval
  Execution halted
```

**Comment by gavinsimpson** [comment]
Thanks @hadley I'll fix this and be ready in time for you updating on CRAN.

**Comment by gavinsimpson** [comment]
Fixed via f8826d1be5d33fbc89b9d58a932343eb4fb183f1
